src/virt-install

```diff
@@ -151,7 +153,15 @@ systemctl daemon-reload
 systemctl start coreos-virtinstall-logs
 %end
     """)
-ks_tmp.flush()
+
+# Support using commits only
+if args.unset_ostree_ref:
+    print("virt-install: Will remove ostree ref")
+    ks_tmp.write("""
+%post --erroronfail
+ostree refs | grep -v '^ostree' | xargs ostree refs --delete
```

Comment:
We already have the ref right? Can't we do `ostree refs --delete {args.ostree_ref}`?

Reply:
It ends up with the ${name} prefix due to anaconda's remote handling.

Reply:
Ahh right, so we can do `ostree refs --delete {args.ostree_remote}:{args.ostree_ref}`? I'm good with this as is too though.

Reply:
Done